cloud-common/src/main/java/com/sequenceiq/cloudbreak/util/BenchMark.java

```diff
@@ -0,0 +1,54 @@
+package com.sequenceiq.cloudbreak.util;
+
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import org.slf4j.Logger;
+
+public class BenchMark {
+    private BenchMark() {
+    }
+
+    public static void b(Runnable callback, Logger logger, String message, Object... params) {
+        long start = System.currentTimeMillis();
+        callback.run();
+        logDuration(logger, message, start, params);
+    }
+
+    public static <T> T b(Supplier<T> callback, Logger logger, String message, Object... params) {
+        long start = System.currentTimeMillis();
+        T resp = callback.get();
+        logDuration(logger, message, start, params);
+        return resp;
+    }
+
+    public static <T, E extends Exception> T cb(SingleCheckedSupplier<T, E> callback, Logger logger, String message, Object... params) throws E {
+        long start = System.currentTimeMillis();
+        T resp = callback.get();
+        logDuration(logger, message, start, params);
+        return resp;
+    }
+
+    public static <T, E extends Exception, X extends Exception> T cb(MultiCheckedSupplier<T, E, X> callback, Logger logger, String message, Object... params)
```

Comment:
What is `cb`? It should be `measure` too, or something like that.

Reply:
yes, cb() is extremely confusing, somehow I immediately associate to Cloudbreak